adafruit_rfm9x.py

```diff
@@ -553,7 +595,12 @@ def spreading_factor(self):
     def spreading_factor(self, val):
         # Set spreading factor (set to 7 to match RadioHead Sf128).
         val = min(max(val, 6), 12)
-        self._write_u8(_RH_RF95_DETECTION_OPTIMIZE, 0xC5 if val == 6 else 0xC3)
+
+        if val == 6:
+            self.detection_optimize = 0x5
```

Comment:
Why aren't we writing to the `_RH_RF95_DETECTION_OPTIMIZE` register here with the value of `detection_optimize`?

Reply:
detection_optimize sets that register.
https://github.com/adafruit/Adafruit_CircuitPython_RFM9x/blob/a375880730b9ac4142b15f754b8123d2587f4d54/adafruit_rfm9x.py#L239

Reply:
This way it does not impact the setting of auto_ifon.

Reply:
I see - OK!